server/handlers/handlers.go

```diff
@@ -41,12 +42,25 @@ const (
 	discardedReasonInactiveUser           = "inactive_user"
 	discardedReasonDuplicatedPost         = "duplicated_post"
 	discardedReasonAlreadyAppliedChange   = "already_applied_change"
+	discardedReasonFileLimitReached       = "file_limit_reached"
+	discardedReasonEmptyFileID            = "empty_file_id"
+	discardedReasonMaxFileSizeExceeded    = "max_file_size_exceeded"
+
+	actionSourceMSTeams    = "msteams"
+	DirectMessageTrue      = "true"
+	DirectMessageFalse     = "false"
+	ActionCreated          = "created"
+	ActionUpdated          = "updated"
+	ActionDeleted          = "deleted"
+	ReactionSetAction      = "set"
+	ReactionUnsetAction    = "unset"
+	IncreaseFileCountByOne = 1
```

Comment:
This variable name is not making sense.

Reply:
yes, because in one case we are incrementing the file count by a custom number rather than 1.